apps/gitness/src/pages-v2/pull-request/__tests__/pull-request-utils.test.ts

```diff
@@ -0,0 +1,253 @@
+import { describe, expect, it } from 'vitest'
```

Comment:
No need to import since they're already set as globals.
```suggestion
```

Reply:
done